### PR TITLE
Local app alpha -> beta

### PR DESCRIPTION
**Pull request by rl-gitpod**

### CODE REVIEW

File: components/supervisor/BUILD.yaml [hunk 1]
```diff
@@ -37,7 +37,8 @@ packages:
       commands:
         - ["curl", "-OL", "https://matt.ucc.asn.au/dropbear/dropbear-2020.81.tar.bz2"]
         - ["tar", "xjf", "dropbear-2020.81.tar.bz2"]
-        - ["sh", "-c", "cd dropbear-2020.81; ./configure --enable-static && make"]
+        - ["sh", "-c", "cd dropbear-2020.81; ./configure --enable-static && sed -i '/clearenv();/d' svr-chansession.c && make"]
```
Comment (akosyakov) [inline components/supervisor/BUILD.yaml:40]: ❤️

Comment (akosyakov) [reply components/supervisor/BUILD.yaml:40]: I am not sure why other changes are required?

Comment (rl-gitpod) [reply components/supervisor/BUILD.yaml:40]: The logging was me 'debugging' - I will remove it prior to making this a non-draft.
The [other](https://github.com/gitpod-io/gitpod/pull/4131/commits/d8380f883175977e275f0992fe9278fa431b9131) is required to get the new dependant packages installed.

Comment (akosyakov) [reply components/supervisor/BUILD.yaml:40]: ok, just ping me when it is good to try

**Pull request closed without merging**